google-dracut-config.spec

```diff
@@ -17,14 +17,16 @@
 
 
 Name:           google-dracut-config
-Version:        0.0.2
+Version:        0.0.3
 Release:        0
 Summary:        Google Dracut config overlay files
 License:        Apache-2.0
 Group:          System/Management
 URL:            https://github.com/SUSE-Enceladus/google-dracut-config
 Source:         %{name}-%{version}.tar.gz
 BuildRequires:  dracut
+Conflicts:      microsoft-dracut-config
+Conflicts:      amazon-dracut-config
```

Comment:
We should do this by using a generic name as provides and the a conflicts with otherproviders see regionServiceClientConfigGCE as an example. I suggest Provides:  dracut-instance-change-config and Conflicts: otherproviders(dracut-instance-change-config)